src/context/data-context.js

```diff
@@ -0,0 +1,45 @@
+import { createContext, useContext, useReducer, useState } from "react";
+import { dataReducer } from "../reducers/dataReducer";
+
+const dataContext = createContext(null);
```

Comment:
D should be caps

Reply:
If I do that I need to changes in all places as its common reducer for all